### PR TITLE
docs: general review of all sections

### PR DESCRIPTION
**Pull request by DamianGlowala**

### CODE REVIEW

**Review by Atinux — APPROVED** [review]

**Comment by Atinux** [comment]
Thanks @DamianGlowala ❤️

File: docs/content/tailwind/1.config.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ This config has less priority over the [tailwind.config.js](#tailwindconfigjs) f
 
 <d-alert type="warning">
 
-This is advanced usage and mostly used for Nuxt modules authors.
+This is an advanced usage section and indended primarily for Nuxt modules authors.
```
Comment (jiblett1000) [inline docs/content/tailwind/1.config.md:101]: typo. should be "Intended"

File: docs/content/tailwind/1.config.md [hunk 2]
```diff
@@ -149,7 +149,7 @@ The `content` option will be:
 ]
 ```
 
-If you want to fully overwrite, you can use a `function` that receives the default value:
+If you want to fully overwrite its value, you can use a `function` that receives the default value:
```
Comment (jiblett1000) [inline docs/content/tailwind/1.config.md:152]: should be it's not its

File: docs/content/tailwind/1.config.md (hunk 1 above)
Comment (Atinux) [reply docs/content/tailwind/1.config.md:101]: Can you open another PR?

Comment (jiblett1000) [reply docs/content/tailwind/1.config.md:101]: Will do :)

**Pull request merged**